virttest/qemu_devices/qcontainer.py

```diff
@@ -1061,16 +1061,26 @@ def pflash_handler(firmware_name, cmd):
                 first_image = images[0]
                 img_params = params.object_params(first_image)
                 img_params["backing_chain"] = "no"
-                img_info = json.loads(qemu_storage.QemuImg(img_params,
-                                                           current_data_dir,
-                                                           first_image).info(True, "json"))
-                img_path, img_name = os.path.split(img_info.get("filename"))
-
-                pflash_vars_name = "%s_%s_VARS.fd" % (self.vmname, img_name)
-                pflash_vars_path = os.path.join(img_path, pflash_vars_name)
-                # When image has backing files, treat it as a temporary image
-                if "backing-filename" in img_info:
-                    self.temporary_image_snapshots.add(pflash_vars_path)
+                img_obj = storage.QemuImg(img_params, current_data_dir, first_image)
+                if img_obj.is_remote_image():
+                    pflash_vars_name = "%s_%s_VARS.fd" % \
+                        (self.vmname, params['image_backend'])
+                    pflash_dir = '/mnt/%s' % params['image_backend']
```

Comment:
May be the avocado tmp dir is better ?

Reply:
But I am worried that /tmp/ FS type is tmpfs will have an impact on the latest version.
Test later

Reply:
Yes,  tmp is no very suitable, my careless,  could you try data_dir ?

Reply:
Ok, the installation of the latest version of the host failed yesterday. Will try it today. Update later.